### PR TITLE
adding dotnet template blog

### PR DESCRIPTION
**Pull request by mark-robustelli**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205443958713398
  - https://app.asana.com/0/0/1205443958779494
  - https://app.asana.com/0/0/1205443958779482
  - https://app.asana.com/0/0/1205443958779506

### CODE REVIEW

**Review by alex-fusionauth — CHANGES_REQUESTED** [review]
Few grammatical thoughts / changes.

Working through the technical side was pretty straight forward, just one call out about the insecure https.

File: astro/src/content/blog/dotnet-templates.mdx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+```shell
+dotnet new install FusionAuth.Templates::1.0.0
+```
+
+That's all there is to obtaining the templates.
+
+### FusionAuth Server
+
+Of course, to effectively use these templates you will need an instance of FusionAuth to bang against. If you already have an instance set up, you likely know the settings you will need. If you do not have an instance to point to, don't fear. You can download an instance and have it up and running in no time assuming you have Docker installed. Simply clone the [fusionauth-example-client-libraries](https://github.com/FusionAuth/fusionauth-example-client-libraries) info, and spin up the Docker instance using the following command.
```
Comment (mark-robustelli) [inline astro/src/content/blog/dotnet-templates.mdx:44]:
```suggestion
Of course, to effectively use these templates you will need an instance of FusionAuth to bang against. If you already have an instance set up, you likely know the settings you will need. If you do not have an instance to point to, don't fear. You can download an instance and have it up and running in no time assuming you have Docker installed. Simply clone the [fusionauth-example-client-libraries](https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/templates) info, and spin up the Docker instance using the following command.
```

File: astro/src/content/blog/dotnet-templates.mdx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+excerpt_separator: "{/* more */}"
+---
+import RemoteCode from '../../components/RemoteCode.astro';
+import Aside from '/src/components/Aside.astro';
+
+Reduce the overhead associated with customer authentication and authorization when creating a .NET web application using FusionAuth templates. 
+{/* more */} 
+
+## Introduction
+*Templates for nothin' and your security for free.*
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/blog/dotnet-templates.mdx:18]: Going for Dire Straits reference.  To obscure?

File: astro/src/content/blog/dotnet-templates.mdx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+import RemoteCode from '../../components/RemoteCode.astro';
+import Aside from '/src/components/Aside.astro';
+
+Reduce the overhead associated with customer authentication and authorization when creating a .NET web application using FusionAuth templates. 
+{/* more */} 
+
+## Introduction
+*Templates for nothin' and your security for free.* 
+
+Literally. The [FusionAuth Templates for Microsoft .NET](https://marketplace.visualstudio.com/items?itemName=FusionAuth.FusionAuthVSTemplates) are free to use and so is the self-hosted [Community Plan](/pricing?step=plan&hosting=self-hosting) of FusionAuth.
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/blog/dotnet-templates.mdx:20]: If I ditch the first line, then I'll alter this.

File: astro/src/content/blog/dotnet-templates.mdx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+It really is just that simple. You now have a fully functioning and secured application. All you need to do is hit the green Start button in Visual Studio. 
+
+Again, if you are not using Visual Studio all you need to do is issue the following command from the same directory the `.csproj` file is in.
+
+```shell
+cd MyFusionAuthTestApplication 
+dotnet run
+```
+
+You can open a browser and go to [https://localhost:5001](https://localhost:5001). Log in using the username `admin@example.com` and the password `password`.
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/blog/dotnet-templates.mdx:91]: @alex-fusionauth , can you run 
`dotnet dev-certs https --check` 
and see if you have the dev-certs installed?  If not you may be able to run
`dotnet dev-certs https --trust`

as described here: https://stackoverflow.com/questions/64017267/dotnet-dev-certs-certificate-not-trusted

If that works, I will add the info.

File: astro/src/content/blog/dotnet-templates.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/src/content/blog/dotnet-templates.mdx:18]: https://www.youtube.com/watch?v=wTP2RUD_cL0

File: astro/src/content/blog/dotnet-templates.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/src/content/blog/dotnet-templates.mdx:91]: did you switch the port to 5000 localhost?

**Pull request merged**